extension-config.json

```
@@ -2,7 +2,63 @@
   "version": "1.0.0",
   "id": "@shopgate-project/swell",
   "components": [
+    {
+      "type": "translations",
+      "path": "frontend/locale/en-US.json",
+      "id": "locale/en-US"
+    },
+    {
+      "id": "NavDrawerItem",
+      "path": "frontend/portals/NavDrawerItem/index.jsx",
+      "target": "nav-menu.home.before",
+      "type": "portals"
+    },
+    {
+      "id": "TabBarItem",
+      "path": "frontend/portals/TabBarItem/index.jsx",
+      "target": "tab-bar.cart.after",
+      "type": "portals"
+    }
   ],
   "configuration": {
+    "showSwellNavigation": {
+      "type": "admin",
+      "destination": "frontend",
+      "default": true,
+      "params": {
+        "label": "Should show navigation button in tab bar and nav drawer",
+        "type": "checkbox"
+      }
+    },
+    "swellButtonText": {
+      "type": "admin",
+      "destination": "frontend",
+      "default": null,
+      "params": {
+        "label": "Swell text for button that leads to Rewards Page",
+        "type": "text",
+        "required": true
+      }
+    },
+    "swellRewardsPath": {
+      "type": "admin",
+      "destination": "frontend",
+      "default": "/rewards",
```

needs to be 
```suggestion
      "default": "/page/rewards",
```
since i will be a cms page

---

Thanks, updated